docs/config.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+  title: "Cosmos Network",
+  description: "Documentation for the Cosmos Network.",
+  dest: "./site-docs",
+  base: "/",
+  markdown: {
+    lineNumbers: true
+  },
+  themeConfig: {
+    lastUpdated: "Last Updated",
+    nav: [{ text: "Back to Cosmos", link: "https://cosmos.network" }],
+    sidebar: [
+      {
+        title: "Introduction",
+        collapsable: false,
+        children: [
+          "/introduction/cosmos-hub",
+          "/introduction/tendermint",
+        ]
+      },
+      {
+        title: "Getting Started",
+        collapsable: false,
+        children: [
+          "/getting-started/voyager",
+          "/getting-started/installation",
+          "/getting-started/full-node",
+          "/getting-started/create-testnet"
+        ]
+      },
+      {
+        title: "Cosmos SDK",
+        collapsable: false,
+        children: [
+          ["/sdk/overview", "Overview"],
+          ["/sdk/core/intro", "Core"],
+          "/sdk/core/app1",
+          "/sdk/core/app2",
+          "/sdk/core/app3",
+          "/sdk/core/app4",
+          "/sdk/core/app5",
+          // "/sdk/modules",
```

Comment:
Why is this commented ?

Reply:
not sure, i copied the original file. Edit as required (that's why it's here)